app/src/main/java/org/oppia/app/player/state/DragDropSortInteractionView.kt

```diff
@@ -26,13 +29,16 @@ import javax.inject.Inject
 class DragDropSortInteractionView @JvmOverloads constructor(
   context: Context, attrs: AttributeSet? = null, defStyleAttr: Int = 0
```

Comment:
ktlint failing for this file, please check with 
`ktlint --android app/src/main/java/org/oppia/app/player/state/DragDropSortInteractionView.kt"`

Reply:
done